scripts/extract-replaygain

```diff
@@ -2,22 +2,42 @@
 
 set -e
 
-FFMPEG=`which ffmpeg || echo ""`
+FFMPEG=`which ffmpeg 2>/dev/null || echo ""`
+FFPROBE=`which ffprobe 2>/dev/null || echo ""`
 
 if test -z "$FFMPEG"; then
-  echo "ffmpeg must be installed"
-  exit 1
+        echo "ffmpeg must be installed"
+        exit 1
 fi
 
-SED=`which gsed || echo ""`
+SED=`which gsed 2>/dev/null || echo ""`
```

Comment:
Do we really want to hide errors here?

Reply:
Aren't these errors  handled by the `if test -z "$SED`; then" line ?